rmf_fleet_adapter_python/src/adapter.cpp

```diff
@@ -166,27 +179,47 @@ PYBIND11_MODULE(rmf_adapter, m) {
     battery::SimpleDevicePowerSink& t_sink,
     double recharge_threshold,
     double recharge_soc,
-    bool account_for_battery_drain)
+    bool account_for_battery_drain,
```

Comment:
Just a comment, not sure should we have a default arg for this. (purely for backward compatibility for others using this api).

Reply:
default arg for `account_for_battery_drain`? That parameter has been there since the previous release. The newly added `finishing_request_string` takes in a default value to make the function call backwards compatible

Reply:
My mistake, overlooked the diff